great_expectations/rule_based_profiler/parameter_builder/parameter_container.py

```diff
@@ -329,26 +313,45 @@ def _get_parameter_value_from_parameter_container(
     fully_qualified_parameter_name: str,
     fully_qualified_parameter_name_as_list: List[str],
     parameter_container: ParameterContainer,
-) -> Optional[Any]:
+) -> Optional[Union[Any, ParameterNode]]:
     parameter_node: Optional[ParameterNode] = parameter_container.get_parameter_node(
         parameter_name_root=fully_qualified_parameter_name_as_list[0]
     )
     if parameter_node is None:
         return None
 
     parameter_name_part: Optional[str] = None
+    return_value: Optional[Union[Any, ParameterNode]] = parameter_node
     try:
         for parameter_name_part in fully_qualified_parameter_name_as_list:
-            if (
-                parameter_node.attributes
-                and parameter_name_part in parameter_node.attributes
-            ):
-                return parameter_node.attributes[parameter_name_part]
+            if parameter_name_part in return_value:
+                return_value = return_value[parameter_name_part]
 
-            parameter_node = parameter_node.descendants[parameter_name_part]
     except KeyError:
         raise KeyError(
             f"""Unable to find value for parameter name "{fully_qualified_parameter_name}": Part \
 "{parameter_name_part}" does not exist in fully-qualified parameter name.
 """
         )
+
+    # TODO: <Alex>ALEX -- leaving the capability below for future considerations.</Alex>
+    # """
+    # Support a shorthand notation (for use in ExpectationConfigurationBuilder): If fully-qualified parameter name does
+    # not end on f"{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY}"
+    # (e.g., ".value") and the "FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY" (e.g., "value") key is available in
+    # "ParameterNode", then return the value, corresponding to the "FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY"
+    # (e.g., "value") key.  Hence, can use shorthand "$parameter.my_min_user_id.column.min" instead of the explicit
+    # "$parameter.my_min_user_id.column.min.value".  Retrieving the details still requires using the explicit
+    # "$parameter.my_min_user_id.column.min.details".
```

Comment:
These examples should change to remove the metric_name e.g. 
```suggestion
    # (e.g., "value") key.  Hence, can use shorthand "$parameter.my_min_user_id" instead of the explicit
    # "$parameter.my_min_user_id.value".  Retrieving the details still requires using the explicit
    # "$parameter.my_min_user_id.details".
```